tests/validators/union.test.ts

```diff
@@ -22,4 +22,75 @@ describe('UnionValidator', () => {
 			])
 		);
 	});
+
+	describe('or', () => {
+		test('Union Or', () => {
```

Comment:
All tests must use `GIVEN ... THEN ...`

Reply:
For validations, we always want to use at least 1 passing and at least 1 failing test, so those should use `describe`, like the optional test in this file.

Reply:
I'm bad at wording can u please update this line?